apps/prairielearn/src/lib/question-render.ts

```diff
@@ -405,6 +405,15 @@ export async function getAndRenderVariant(
   options?: {
     urlOverrides?: Partial<QuestionUrls>;
     publicQuestionPreview?: boolean;
+    /**
+     * Whether or not any recorded issues should have their extra data loaded.
+     * If not specified, the default is to load extra data if we're in dev mode
+     * or if the user has permission to view course data.
+     *
+     * This toggle is useful mainly for AI question generation, where we always
+     * want to load issue data so we can provided it as context to the model.
+     */
+    issuesLoadExtraData?: boolean;
```

Comment:
Not blocking, but I generally prefer options to have clear default values, and not use `undefined` to mean something different to the default. In general I think code is easier to understand when we have explicit defaults like
```
{ a = 1, b = 2 }: { a?: number; b?: number } = {}
```

Reply:
Done in 7ac54be. Look good?

Reply:
Nice! 🫛